src/main/java/cl/transbank/pos/SDKTest.java

```diff
@@ -0,0 +1,104 @@
+package cl.transbank.pos;
```

Comment:
If this is the file you use to test while you code, it doesn't belong to the SDK. The way we provide to test the sdk is through the example project.

Reply:
I will remove it.